nixos/modules/services/security/hologram-server.nix

```diff
@@ -94,9 +96,15 @@ in {
       after       = [ "network.target" ];
       wantedBy    = [ "multi-user.target" ];
 
-      serviceConfig = {
-        ExecStart = "${pkgs.hologram.bin}/bin/hologram-server --debug --conf ${cfgFile}";
-      };
+      inherit script;
+    };
+
+    docker-containers.hologram-server = {
```

Comment:
We could probably abstract many basic simple services like this with a simple function to automatically do `docker-containers`, `trivial-services`, and maybe even `systemd.services`. The autogenerated config could then be merged with idiosyncratic config for any of those.